tests/ttnn/unit_tests/operations/test_new_conv2d.py

```diff
@@ -370,12 +372,14 @@ def test_conv_ws(
     stride_h = stride
     stride_w = stride
     batch_size = 2
-    fp32_accum = False
-    packer_l1_acc = False
+    fp32_accum = True
+    packer_l1_acc = True
     deallocate_activation = False
     debug = False
     groups = 1
 
+    if input_channels == 1280 and output_channels == 1280 and input_height == 16:
```

Comment:
pls added a comment stating todo and issue that is supposed to resolve this case, and enable the test case

Reply:
Done.